elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -1393,6 +1394,23 @@ public void createChildRelateExisting() {
                 .body(equalTo(expected));
     }
 
+    @Test
+    public void invalidPatchMissingId() {
+        String request = jsonParser.getJson("/ResourceIT/invalidPatchMissingId.req.json");
+
+        String detail = "Bad Request Body'Patch extension requires all objects to have an assigned "
+                + "ID (temporary or permanent) when assigning relationships.'";
+
+        given()
+                .contentType(JSONAPI_CONTENT_TYPE_WITH_JSON_PATCH_EXTENSION)
+                .accept(JSONAPI_CONTENT_TYPE_WITH_JSON_PATCH_EXTENSION)
+                .body(request)
+                .patch("/")
+                .then()
+                .statusCode(HttpStatus.SC_BAD_REQUEST)
+                .body("errors[0].detail[0]", equalTo(Encode.forHtml(detail)));
```

Comment:
@aklish any idea why detail is now treated as an array here?

Reply:
That might be a bug.  I checked JSON-API spec - and I think that should just be a string.

Reply:
Yes, I suspect it might be processed as
`(errors[0].detail)[0]` due to the outer and inner arrays.